lib/rules/attributes-order.js

```diff
@@ -200,17 +206,27 @@ function create(context) {
     [ATTRS.UNIQUE, ATTRS.SLOT],
     ATTRS.TWO_WAY_BINDING,
     ATTRS.OTHER_DIRECTIVES,
-    ATTRS.OTHER_ATTR,
+    [ATTRS.OTHER_ATTR_DYNAMIC, ATTRS.OTHER_ATTR_STATIC],
     ATTRS.EVENTS,
     ATTRS.CONTENT
   ]
   if (context.options[0] && context.options[0].order) {
-    attributeOrder = context.options[0].order
+    attributeOrder = [...context.options[0].order]
```

Comment:
Do you think it makes sense to add a validation check here, so that `OTHER_ATTR` should not be specified together with either `OTHER_ATTR_DYNAMIC` or `OTHER_ATTR_STATIC`? Or actually, that no category should appear twice in the options?

Reply:
Fixed, it really got better